async-development/package.json

```
@@ -4,7 +4,7 @@
   "description": "",
   "main": "main.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
+    "test": "mocha js/**/*.test.js --require @babel/register --require mock-local-storage",
     "bundle": "browserify ./js/main.js -o ./dist/bundle.js -t [ babelify --presets [ @babel/preset-env ] --plugins [ @babel/transform-runtime ] ]"
```

babel configs could be removed now since they are defined in `.babelrc`
```suggestion
    "bundle": "browserify ./js/main.js -o ./dist/bundle.js -t [ babelify ]"
```